docs/install/osx_setup.md

```diff
@@ -100,14 +104,21 @@ After you have installed the dependencies, pull the MXNet source code from Git a
 The file called ```osx.mk``` has the configuration required for building MXNet on OS X. First copy ```make/osx.mk``` into ```config.mk```, which is used by the ```make``` command:
 
 ```bash
-    git clone --recursive https://github.com/apache/incubator-mxnet ~/mxnet
-    cd ~/mxnet
-    cp make/osx.mk ./config.mk
-    echo "USE_BLAS = openblas" >> ./config.mk
-    echo "ADD_CFLAGS += -I/usr/local/opt/openblas/include" >> ./config.mk
-    echo "ADD_LDFLAGS += -L/usr/local/opt/openblas/lib" >> ./config.mk
-    echo "ADD_LDFLAGS += -L/usr/local/lib/graphviz/" >> ./config.mk
-    make -j$(sysctl -n hw.ncpu)
+git clone --recursive https://github.com/apache/incubator-mxnet ~/mxnet
+cd ~/mxnet
+mkdir build && cd build
+cmake \
+    -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+    -DCMAKE_C_COMPILER_LAUNCHER=ccache \
```

Comment:
Can we make this options default in the `cmake_options_osx.yml` so users can run the simplest `cmake` to build MXNet just like we are running `make`

Reply:
cmake_options_osx.yml is used by dev_menu.py not by cmake.  I can remove ccache if it bothers you so much.